packages/dds/map/src/mapKernel.ts

```diff
@@ -604,11 +636,11 @@ export class MapKernel {
                 },
                 submit: (op: IMapDeleteOperation, localOpMetadata: unknown) => {
                     // We don't reuse the metadata but send a new one on each submit.
-                    this.submitMapKeyMessage(op);
+                    this.submitMapKeyMessage(op, localOpMetadata);
                 },
                 getStashedOpLocalMetadata: (op: IMapDeleteOperation) => {
```

Comment:
I don't know when we stash. Is it possible to coincide with rollback or rollback-able ops?

Reply:
i don't think so, but maybe @wes-carlson  can confirm. i believe stashed ops will have always made to the delta manager,  whereas for rollback the will have never made it further than the pending state manager of the container runtime. i wonder if we can test, or enforce this constraint

Reply:
Pending ops can be generated and stashed while disconnected, in which case they will not have made it to delta manager

Reply:
That could be problematic. There's not enough information in the op to uniquely match it to the right localOpMetadata now. Is there a reason why applyStashedOp doesn't have the localOpMetadata unlike the other IDeltaHandler methods? I'm guessing we can't assume it must be the most recent clear or edit to the key, which is what my code is doing.

Reply:
we can't serialize localOpMetadata. i think we should prevent stashed ops from persisting partial batches

Reply:
Changed rollback data to undefined for stashed ops and rollback throws error if it sees this. There's a separate task open for preventing stashed ops during orderSequentially.